JustSaying/AwsTools/MessageHandling/SqsPublisher.cs

```diff
@@ -46,10 +56,25 @@ public void Publish(Message message)
         public async Task PublishAsync(Message message, CancellationToken cancellationToken)
```

Comment:
Why do we not return the response? We don't like consumers to interact with AWS?

Reply:
@JosephWoodward asked this question when starting the work.

I don't like it for 2 reasons.

First it's not an intuitive fit for the API, there is something very elegant and intuitive about a Publish method that is effectively a command.

Second, JustSaying tries not to expose AWSSDK types where possible, it is an abstraction, rather than a composable toolkit. There is an argument to be made to favour a composable toolkit, however this isn't what the library is, so we should stay with it IMO.

Reply:
I agree with your second comment about what JustSaying _is_. Can have the discussion about which is preferable some other time.